### PR TITLE
add CheckStyle-IDEA Plugin introduction

### PR DESCRIPTION
**Pull request by lhyundeadsoul**
CheckStyle-IDEA Plugin

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: docs/en/contribution/setup.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ See [install plugins for IDEA](https://www.jetbrains.com/help/idea/managing-plug
 Before running the following example, you should also install JetBrains IntelliJ IDEA's [Lombok plugin](https://plugins.jetbrains.com/plugin/6317-lombok).
 See [install plugins for IDEA](https://www.jetbrains.com/help/idea/managing-plugins.html#install-plugins) if you want to.
 
+### Install JetBrains IDEA CheckStyle-IDEA Plugin
+
+Before coding, you should also install JetBrains IntelliJ IDEA's [CheckStyle-IDEA plugin](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea).
+See [install plugins for IDEA](https://www.jetbrains.com/help/idea/managing-plugins.html#install-plugins) if you want to.
+Next, you should go to `Preferences -> Editor -> Code style -> Scheme -> Import Scheme -> CheckStyle Configration` and import `tools/checkstyle/checkStyle.xml`
```
Comment (CalvinKirs) [inline docs/en/contribution/setup.md:53]: Preferences -> Editor -> Code Style -> Java
If the user wants to change to automatic formatting, these configurations are also required

Comment (lhyundeadsoul) [reply docs/en/contribution/setup.md:53]: Do you mean I should add `If the user wants to change to automatic formatting, these configurations are also required`  to the doc?

Comment (CalvinKirs) [reply docs/en/contribution/setup.md:53]: yes

**Comment by ic4y** [comment]
checkstyle also needs to be configured
image

**Comment by lhyundeadsoul** [comment]
> checkstyle also needs to be configured image

I haven't configured this, but it works. Is it essential?

**Comment by ic4y** [comment]
> > checkstyle also needs to be configured image
> 
> I haven't configured this, but it works. Is it essential?

 I think so.

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**